.github/autobump.txt

```diff
@@ -232,9 +232,9 @@ bats-core
 batt
 bazarr
 bazel
-bazel@7
 bazel-diff
 bazel-remote
+bazel@7
```

Comment:
This change doesn't look intentional here.

Reply:
this is sorted now.

Reply:
actually it got resolved in master, rebased the pr now.